var/spack/repos/builtin/packages/py-pbr/package.py

```diff
@@ -34,3 +34,4 @@ class PyPbr(PythonPackage):
     version('1.8.1', 'c8f9285e1a4ca6f9654c529b158baa3a')
 
     depends_on('py-setuptools', type='build')
+    depends_on('py-enum34', type='build')
```

Comment:
It looks like `py-pbr` does not work on all versions of Python; or at least, `py-enum34` is only required for certain older Python versions.  Please indicate --- not just for this package but for all --- what versions of Python it works with.  If it is Python2-only, add `depends_on('python@2:2.8')`.  Otherwise, add a comment instead saying `# This works with Python2 and Python3`.